src/zc/FileStorage/tests.py

```diff
@@ -177,15 +177,15 @@ def test_pack_sleep():
     ...        break
     ...     time.sleep(0.01)
     >>> def faux_sleep(x):
-    ...     print 'sleep '+`x`
+    ...     print('sleep '+repr(x))
     >>> time.sleep = faux_sleep
     >>> conn.root().x = 1
     >>> transaction.commit()
     >>> thread.join()
     sleep 1.0
 
     >>> fs.close()
-    >>> print open('data.fs.packlog').read(), # doctest: +NORMALIZE_WHITESPACE
+    >>> print (open('data.fs.packlog').read(),) # doctest: +NORMALIZE_WHITESPACE
```

Comment:
You can drop the trailing comma, or replace it with a `.rstrip('\n')`.

Reply:
Also, this looks like you're trying to print a tuple?  Which is not what the test expects on Python 2, and not how the print function acts on Python 3?  So maybe drop the space in front of the `(`?